packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md

```diff
@@ -7,6 +7,52 @@ section: get-started
 import './get-started.css';
 import { Divider, Timestamp } from '@patternfly/react-core';
 
+<Timestamp>December 2024</Timestamp>
+
+## PatternFly 6.1
+
+### Promoted package versions
+
+### Component updates 
+
+- We fixed bugs that were reported in a few React components. More details are available in [the patternfly-react release notes](https://github.com/patternfly/patternfly-react/releases). 
+
+
+### Website updates 
+
+- We fixed an issue on PatternFly.org that prevented proper page loading when navigating to new pages via in-content links.
+
+- We updated [the PatternFly.org accessibility scorecard](/accessibility/accessibility-scorecard#patternfly.org-accessibility-scorecard). We will begin working on any accessibility violations that were identified.
+
+#### Documentation updates
+
+- We updated design guideline images for [context selector](/components/menus/context-selector/design-guidelines) to align with PatternFly 6 styles.  
+- We updated and added "draggable" examples to [dual list selector](/components/dual-list-selector#draggable) and [ data list](/components/data-list#draggable).
+- We updated our ChatBot documentation: 
+    - We renamed the extension to "ChatBot", to better brand our offering and help distinguish it from other general "chatbot" features.
+    - We combined and organized our documentation into a few cohesive pages: ["Overview"](/patternfly-ai/chatbot/overview), ["UI"](/patternfly-ai/chatbot/ui), and ["Messages"](/patternfly-ai/chatbot/message). Note that our documentation structure could continue to change as our extension grows. We will continue to share updates when changes occur. 
+    - We moved our [basic and embedded ChatBot demos](/patternfly-ai/chatbot/overview/demo) to the "Overview" page for better visibility. 
+    - We added initial [ChatBot design guidelines](/patternfly-ai/chatbot/overview/design-guidelines), which can also be found on the "Overview" page.
+    - We added documentation for new features, as described in the following section. 
+
+### Extension updates 
+
+#### ChatBot
+
+We've continued to evolve our new ChatBot:
+- We renamed the repo from "patternfly/visual-assistant" to "patternfly/chatbot" to align with our website docs.
+- We added support for new features, including: 
+    - [Multiple file attachments](/patternfly-ai/chatbot/messages#messages-with-attachments). 
+    - [Quick response buttons](/patternfly-ai/chatbot/messages#messages-with-quick-responses). 
+    - Additional customization options for [welcome messages](/patternfly-ai/chatbot/ui#welcome-message), [message avatars](/patternfly-ai/chatbot/messages#messages), [message actions](/patternfly-ai/chatbot/messages#custom-messages-actions), [modal](/patternfly-ai/chatbot/ui#modal), [toggle shape](/patternfly-ai/chatbot/ui#custom-toggle-shape).
+    - A variety of styling updates to shadows, sizing, and positioning. 
```

Comment:
We've also added these since the last release:
- Guidance on how to skip to a chatbot (for accessibility purposes)
- Swapped out the textarea in the message bar for a custom one - previous one was an inappropriate library created by some random college student (I didn't pick the original one, I swear!)
- Truncate Message bot/user names if they're too long
- Handle more complex list output in messages 
- Added stop button option for message bar

Reply:
The avatar and toggle changes are not marked as release blockers, so they may not make it depending on when the freeze is called.

Reply:
perfect ty!

Reply:
We just merged the avatars - it includes a breaking change for the tooltip on the chatbot toggle: https://github.com/patternfly/chatbot/pull/335. We changed it from toolTipLabel to tooltipLabel to match the other props.

Reply:
This looks perfect; thank you!